templates/mixplank/next/src/app/data.ts

```diff
@@ -1,18 +1,21 @@
 import { createClient } from "@clickhouse/client-web"
```

Comment:
is this running on the server or in the client? thinking about in prod where this might not have network access to the db from the client

Reply:
Good call, will use server side instead